tests/e2e/specs/shopper/checkout-different-addresses.test.js

```diff
@@ -0,0 +1,101 @@
+/**
+ * External dependencies
+ */
+import {
+	merchant,
+	openDocumentSettingsSidebar,
+	uiUnblocked,
+} from '@woocommerce/e2e-utils';
+
+import {
+	visitBlockPage,
+	selectBlockByName,
+	saveOrPublish,
+} from '@woocommerce/blocks-test-utils';
+
+/**
+ * Internal dependencies
+ */
+import {
+	getNormalPagePermalink,
+	shopper,
+	visitPostOfType,
+} from '../../../utils';
+
+const config = require( 'config' );
+
+const block = {
+	name: 'Checkout',
+};
+
+const simpleProductName = '128GB USB Stick';
+const pageName = 'Checkout Block';
+
+if ( process.env.WOOCOMMERCE_BLOCKS_PHASE < 2 )
+	// eslint-disable-next-line jest/no-focused-tests
+	test.only( `skipping ${ block.name } tests`, () => {} );
+
+describe( `${ block.name } Block (frontend)`, () => {
```

Comment:
Let's change this to something specific to what is being tested. 
`Shopper → Checkout → Can have different shipping and billing addresses` could work here

Reply:
Great suggestion. I changed it accordingly.